contrib/kind.yaml.j2

```diff
@@ -54,7 +54,7 @@ nodes:
          authorization-mode: "AlwaysAllow"
 {%- if ovn_ha is equalto "true" %}
 {%- for _ in range(1, ovn_num_master | int) %}
- - role: worker
+ - role: control-plane
```

Comment:
this is not needed, one thing is kubernets HA and other OVN HA

I´m not familiar with the haMode var in the tests, but a quick look, it seems that is only used to identify the nodes because they were using different hostnames, and is not related to any HA test.

Reply:
As discussed over the slack, hostname are used to determine if it's running in ha mode or not. Based on that it decides one which node it needs to pick up. But even if it fails in deciding the hamode, test still works, that's a good and bad news both :).